# Translations template for python-fedora.
# Copyright (C) 2011 ORGANIZATION
# This file is distributed under the same license as the python-fedora
# project.
# 
# Translators:
# Kévin Raymond <shaiton@fedoraproject.org>, 2011.
msgid ""
msgstr ""
"Project-Id-Version: python-fedora\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2011-09-03 16:15-0700\n"
"PO-Revision-Date: 2011-11-10 12:20+0000\n"
"Last-Translator: shaiton <shaiton@fedoraproject.org>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.6\n"
"Language: fr\n"
"Plural-Forms: nplurals=2; plural=(n > 1)\n"

#: fedora/iterutils.py:36
msgid "fedora.iterutils is deprecated.  Use kitchen.iterutils instead"
msgstr "fedora.iterutils est obsolète. Utilisez kitchen.iterutils"

#: fedora/iterutils.py:54
msgid ""
"fedora.iterutils.isiterable is deprecated, use kitchen.iterutils.isiterable "
"instead"
msgstr ""
"fedora.iterutils.isiterable est obsolète. Utilisez "
"kitchen.iterutils.isiterable"

#: fedora/release.py:9
msgid "Python modules for interacting with Fedora Services"
msgstr "Modules python pour interagir avec les services de Fedora"

#: fedora/release.py:10
msgid ""
"\n"
"The Fedora Project runs many different services.  These services help us to\n"
"package software, develop new programs, and generally put together the distro.\n"
"This package contains software that helps us do that.\n"
msgstr ""
"\n"
"Le Projet Fedora gère de nombreux services différents. Ces services nous aident à\n"
"empaqueter des logiciels, développer de nouveaux programmes et généralement à mettre en place la distribution.\n"
"Ce paquet contient les logiciels qui nous aideront à le faire.\n"

#: fedora/textutils.py:33
msgid "fedora.textutils is deprecated.  Use kitchen.text.converters instead"
msgstr "fedora.textutils est obsolète. Utilisez kitchen.text.converters"

#: fedora/accounts/fas2.py:8
msgid ""
"fedora.accounts.fas2 has moved to fedora.client.fas2.  This location will "
"disappear in 0.4"
msgstr ""
"fedora.accounts.fas2 est remplacé par fedora.client.fas2. Cet objet "
"disparaîtra dans la 0.4"

#: fedora/client/__init__.py:89
msgid ""
"DictContainer is deprecated.  Use the Bunch class from python-bunch instead."
msgstr ""
"DictContainer est obsolète. Utilisez la classe Bunch provenant de python-"
"bunch."

#: fedora/client/baseclient.py:94 fedora/client/baseclient.py:215
#: fedora/client/baseclient.py:234 fedora/client/baseclient.py:246
msgid "session_cookie is deprecated, use session_id instead"
msgstr "session_cookie est obsolète. Utilisez session_id"

#: fedora/client/baseclient.py:112
#, python-format
msgid "Unable to load session from %(file)s"
msgstr "Impossible de charger la session à partir de %(file)s"

#: fedora/client/baseclient.py:129
#, python-format
msgid "Unable to create %(dir)s: %(error)s"
msgstr "Impossible de créer %(dir)s : %(error)s"

#: fedora/client/baseclient.py:141
#, python-format
msgid "Unable to write to session file %(session)s: %(error)s"
msgstr "Impossible d'écrire le fichier de session %(session)s : %(error)s"

#: fedora/client/baseclient.py:164
#, python-format
msgid "No session cached for \"%s\""
msgstr "Pas de cache de session pour « %s »"

#: fedora/client/baseclient.py:301
#, python-format
msgid "send_request() got an unexpected keyword argument \"%(arg)s\""
msgstr "send_request() a reçu un argument inattendu « %(arg)s »"

#: fedora/client/baseclient.py:305
msgid "send_request() got an unexpected keyword argument \"input\""
msgstr "send_request() a reçu un argument inattendu « input »"

#: fedora/client/baseclient.py:309
msgid "send_request() got an unexpected keyword argument"
msgstr "send_request() a reçu un argument inattendu"

#: fedora/client/baseclient.py:313
msgid ""
"send_request(input) is deprecated.  Use send_request(req_params) instead"
msgstr "send_request(input) est obsolète. Utilisez send_request(req_params)"

#: fedora/client/baseclient.py:331
msgid ""
"Auth was requested but no way to perform auth was given.  Please set "
"username and password or session_id before calling this function with "
"auth=True"
msgstr ""
"Auth est requis mais aucun moyen d'effectuer l'auth n'est donnée. Merci de "
"définir un nom d'utilisateur et un mot de passe ou un id de session avant "
"d'appeler cette fonction avec auth=True"

#: fedora/client/bodhi.py:195
msgid "You must specify a username"
msgstr "Vous devez donner un nom d'utilisateur"

#: fedora/client/bodhi.py:265
#, python-format
msgid "Reading from %s "
msgstr "Lecture à partir de %s"

#: fedora/client/fas2.py:231
#, python-format
msgid "FAS server unable to retrieve group %(group)s"
msgstr "Groupe %(group)s introuvable dans le serveur FAS"

#: fedora/client/fas2.py:363
msgid "key must be one of \"id\", \"username\", or \"email\""
msgstr "La clef doit être parmi « id », « username », ou « email »"

#: fedora/client/fas2.py:370
#, python-format
msgid "%(field)s is not a valid field to filter"
msgstr "%(field)s n'est pas un champ valide pour trier"

#: fedora/client/fas2.py:422
msgid ""
"people_by_id() is deprecated and will be removed in 0.4.  Please port your "
"code to use people_by_key(key='id', fields=['human_name', 'email', "
"'username', 'bugzilla_email']) instead"
msgstr ""
"people_by_id() est obsolète et sera supprimé en 0.4. Merci d'utiliser dans "
"votre code people_by_key(key='id', fields=['human_name', 'email', "
"'username', 'bugzilla_email'])"

#: fedora/client/fas2.py:604
msgid "FAS server unable to retrieve group members"
msgstr "Le serveur FAS n'a pas pu trouver les membres du groupe"

#: fedora/client/fas2.py:626
msgid "FAS server unable to retrieve user information"
msgstr "Le serveur FAS n'a pas pu trouver les informations utilisateur"

#: fedora/client/fasproxy.py:67
msgid ""
"FasProxyClient() got an unexpected keyword argument 'session_as_cookie'"
msgstr "FasProxyClient() a reçu un argument inattendu 'session_as_cookie'"

#: fedora/client/pkgdb.py:197
#, python-format
msgid ""
"We do not have enough information to create package %(pkg)s. Need version "
"owner."
msgstr ""
"Nous n'avons pas suffisamment d'informations pour créer le paquet %(pkg)s. "
"Nous avons besoin du propriétaire actuel."

#: fedora/client/pkgdb.py:208
#, python-format
msgid "PackageDB returned an error creating %(pkg)s: %(msg)s"
msgstr "PackageDB a retourné une erreur en créant %(pkg)s : %(msg)s"

#: fedora/client/pkgdb.py:231 fedora/client/pkgdb.py:287
#, python-format
msgid "Unable to save all information for %(pkg)s: %(msg)s"
msgstr "Impossible de sauver toutes les informations pour %(pkg)s : %(msg)s"

#: fedora/client/pkgdb.py:323
#, python-format
msgid ""
"Collection abbreviation %(collection)s is unknown.  Use F, FC, EL, or OLPC"
msgstr ""
"L'abréviation collection %(collection)s est inconnue. Utilisez F, FC, EL ou "
"OLPC"

#: fedora/client/pkgdb.py:350
msgid ""
"collection and collection_ver are deprecated names for collctn_name and "
"collctn_ver respectively.  Ignoring the values given in them."
msgstr ""
"collection et collection_ver sont obsolètees pour collctn_name et "
"collctn_ver, respectivement. Les valeurs entrées sont ignorées."

#: fedora/client/pkgdb.py:356
msgid ""
"collection has been renamed to collctn_name.\n"
"Please start using the new name.  collection will go away in 0.4.x."
msgstr ""
"collection a été renommé en collctn_name.\n"
"Merci d'utiliser ce nouveau nom. collection sera supprimé en 0.4.x."

#: fedora/client/pkgdb.py:362
msgid ""
"collection_ver has been renamed to collctn_ver.\n"
"Please start using the new name.  collection_ver will go away in 0.4.x."
msgstr ""
"collection_ver a été renommé en collctn_ver.\n"
"Merci d'utiliser ce nouveau nom. collection sera supprimé en 0.4.x."

#: fedora/client/pkgdb.py:399
msgid ""
"collectn_list is a deprecated name for collctn_list.\n"
"Ignoring the value of collectn_list."
msgstr ""
"collectn_list a été renommé en collctn_list.\n"
"La valeur de collectn_list est ignorée."

#: fedora/client/pkgdb.py:404
msgid ""
"collectn_list has been renamed to collctn_list.\n"
"Please start using the new name.  collectn_list will go away in 0.4.x."
msgstr ""
"collection_list a été renommé en collctn_list.\n"
"Merci d'utiliser ce nouveau nom.  collection_list sera supprimé en 0.4.x."

#: fedora/client/pkgdb.py:478
msgid ""
"collectn is a deprecated name for collctn.\n"
"Ignoring the value of collectn."
msgstr ""
"collectn est un mot obsolète pour collctn.\n"
"La valeur de collectn est ignorée."

#: fedora/client/pkgdb.py:483
msgid ""
"collectn has been renamed to collctn.\n"
"Please start using the new name.  collectn will go away in 0.4.x."
msgstr ""
"collectn a été renommé en collctn.\n"
"Merci d'utiliser ce nouveau nom. collectn sera supprimé en 0.4.x."

#: fedora/client/pkgdb.py:493
#, python-format
msgid "Collection shortname %(collctn)s is unknown."
msgstr "Le nom abrégé collection %(collctn)s est inconnu."

#: fedora/client/proxyclient.py:151
msgid "proxyclient.__init__:entered"
msgstr "proxyclient.__init__:entered"

#: fedora/client/proxyclient.py:160
msgid ""
"Returning cookies from send_request() is deprecated and will be removed in "
"0.4.  Please port your code to use a session_id instead by calling the "
"ProxyClient constructor with session_as_cookie=False"
msgstr ""
"Les cookies renvoyés par send_request() sont obsolètes et seront supprimés "
"en 0.4. Merci d'utiliser dans votre code session_id en faisant appel au "
"constructeur ProxyClient avec session_as_cookie=False"

#: fedora/client/proxyclient.py:167
msgid "proxyclient.__init__:exited"
msgstr "proxyclient.__init__:exited"

#: fedora/client/proxyclient.py:254
msgid "proxyclient.send_request: entered"
msgstr "proxyclient.send_request: entered"

#: fedora/client/proxyclient.py:263
msgid ""
"Giving a cookie to send_request() to authenticate is deprecated and will be "
"removed in 0.4. Please port your code to use session_id instead."
msgstr ""
"Faire passer les cookies d'authentification par send_request() est obsolète "
"et sera supprimé en 0.4. Merci d'utiliser dans votre code session_id."

#: fedora/client/proxyclient.py:273
msgid "username and password must both be set in auth_params"
msgstr ""
"Le nom d'utilisateur et le mot de passe doivent être définis dans "
"auth_params"

#: fedora/client/proxyclient.py:276
msgid ""
"No known authentication methods specified: set \"cookie\" in auth_params or "
"set both username and password in auth_params"
msgstr ""
"Aucune méthode d'authentification spécifiée : définir un « cookie » dans "
"auth_params ou définir le nom d'utilisateur et le mot de passe dans "
"auth_params"

#: fedora/client/proxyclient.py:365
#, python-format
msgid "Creating request %(url)s"
msgstr "Création de la requête %(url)s"

#: fedora/client/proxyclient.py:367
#, python-format
msgid "Headers: %(header)s"
msgstr "Entêtes : %(header)s"

#: fedora/client/proxyclient.py:372
#, python-format
msgid "Data: %(data)s"
msgstr "Données : %(data)s"

#: fedora/client/proxyclient.py:389
msgid "Authentication failed logging in"
msgstr "Échec de l'authentification"

#: fedora/client/proxyclient.py:390
msgid ""
"Unable to log into server.  Invalid authentication tokens.  Send new "
"username and password"
msgstr ""
"Impossible de se connecter au serveur. Jetons d'authentification invalides. "
"Envoyer un nouveau nom d'utilisateur et mot de passe"

#: fedora/client/proxyclient.py:396
#, python-format
msgid "Attempt #%(try)s failed"
msgstr "Essai #%(try)s échoué"

#: fedora/client/proxyclient.py:403
msgid "Unknown HTTP Server Response"
msgstr "Réponse du serveur HTTP inconnue"

#: fedora/client/proxyclient.py:425
#, python-format
msgid "Error returned from json module while processing %(url)s: %(err)s"
msgstr ""
"Une erreur est apparue dans le module json en analysant %(url)s : %(err)s"

#: fedora/client/proxyclient.py:441
msgid "proxyclient.send_request: exited"
msgstr "proxyclient.send_request: exited"

#: fedora/client/wiki.py:67
#, python-format
msgid "Login failed: %(data)s"
msgstr "Connexion échouée : %(data)s"

#: fedora/client/wiki.py:89
#, python-format
msgid "From %(then)s to %(now)s"
msgstr "De %(then)s à %(now)s"

#: fedora/client/wiki.py:92
#, python-format
msgid "%d wiki changes in the past week"
msgstr "%d changements du wiki cette semaine"

#: fedora/client/wiki.py:94
msgid ""
"Warning: Number of changes reaches the API return limit.\n"
"You will not get the complete list of changes unless\n"
"you run this script using a 'bot' account."
msgstr ""
"Attention : la limite de retour de l'API est atteinte.\n"
"Vous n'aurez pas la liste complète des modifications sauf en\n"
"exécutant ce script avec un compte « bot »."

#: fedora/client/wiki.py:104
msgid ""
"\n"
"== Most active wiki users =="
msgstr ""
"\n"
"== Utilisateurs du wiki les plus actifs =="

#: fedora/client/wiki.py:111
msgid ""
"\n"
"== Most edited pages =="
msgstr ""
"\n"
"== Pages les plus éditées =="

#: fedora/compat25/__init__.py:9
msgid ""
"fedora.pycompat25.defaultdict is deprecated, use "
"kitchen.pycompat25.collections.defaultdict instead"
msgstr ""
"fedora.pycompat25.defaultdict est obsolète. Utilisez "
"kitchen.pycompat25.collections.defaultdict"

#: fedora/django/auth/models.py:50
msgid "Loading FAS groups..."
msgstr "Chargement des groupes FAS..."

#: fedora/django/auth/models.py:56
msgid "Unable to load FAS groups. Did you set FAS_USERNAME and FAS_PASSWORD?"
msgstr ""
"Impossible de charger les groupes FAS. Avez-vous défini FAS_USERNAME et "
"FAS_PASSWORD ?"

#: fedora/django/auth/models.py:63
msgid ""
"FAS groups loaded. Don't forget to set FAS_USERNAME and FAS_PASSWORD to a "
"low-privilege account."
msgstr ""
"Groupes FAS chargés. N'oubliez pas de définir FAS_USERNAME et FAS_PASSWORD "
"sur un compte à faibles privilèges."

#: fedora/tg/client.py:8
msgid ""
"fedora.tg.client has moved to fedora.client.  This location will disappear "
"in 0.4"
msgstr ""
"fedora.tg.client est devenu fedora.client. Cette définition sera supprimée "
"en 0.4"

#: fedora/tg/controllers.py:60
#, python-format
msgid "Welcome, %s"
msgstr "Bienvenue %s"

#: fedora/tg/controllers.py:69
msgid ""
"The credentials you supplied were not correct or did not grant access to "
"this resource."
msgstr ""
"Les accréditations fournies ne sont pas correctes ou ne vous permettent pas "
"l'accès à cette ressource."

#: fedora/tg/controllers.py:72
msgid "You must provide your credentials before accessing this resource."
msgstr ""
"Vous devez fournir les accréditations avant d'accéder à cette ressource."

#: fedora/tg/controllers.py:75
msgid "Please log in."
msgstr "Veuillez vous connecter."

#: fedora/tg/controllers.py:100
msgid "You have successfully logged out."
msgstr "Vous vous êtes déconnecté avec succès."

#: fedora/tg/tg1utils.py:38
msgid ""
"fedora.tg.tg1utils is deprecated.  Switch to fedora.tg.utils instead.  This "
"file will disappear in 0.4"
msgstr ""
"fedora.tg.tg1utils est obsolète. Utilisez plutôt fedora.tg.utils. Ce fichier"
" disparaîtra avec la 0.4"

#: fedora/tg/tg2utils.py:38
msgid ""
"fedora.tg.tg2utils is deprecated.  Switch to fedora.tg2.utils instead.  This"
" file will disappear in 0.4"
msgstr ""
"fedora.tg.tg2utils est obsolète. Utilisez plutôt fedora.tg2.utils. Ce "
"fichier disparaîtra avec la 0.4"

#: fedora/tg/util.py:40
msgid ""
"fedora.tg.util is deprecated.  Switch to one of these instead:   TG1 apps: "
"fedora.tg.utils   TG2 apps: fedora.tg2.utils.  This file will disappear in "
"0.4"
msgstr ""
"fedora.tg.util est obsolète. Utilisez plutôt un de ceux-ci : Applications "
"TG1 : fedora.tg.utilsApplications TG2 : fedora.tg2.utils.Ce fichier "
"disparaîtra avec la 0.4"

#: fedora/tg/utils.py:102
msgid "url() expects a dictionary for query parameters"
msgstr "url() attend un dictionnaire comme paramètre de recherche"

#: fedora/tg/identity/jsonfasprovider1.py:223
#: fedora/tg/identity/jsonfasprovider2.py:404
#, python-format
msgid "Error logging in %(user)s: %(error)s"
msgstr "Erreur de connexion pour %(user)s : %(error)s"

#: fedora/tg/identity/jsonfasprovider2.py:56
msgid ""
"Cannot enable ssl certificate auth via identity.ssl without setting "
"fas.usernamme and fas.password for authorization"
msgstr ""
"Le certificat d'authentification ssl ne peut être établi avec identity.ssl "
"sans définir les autorisations fas.usernamme ni fas.password"

#: fedora/tg/identity/jsonfasprovider2.py:166
#, python-format
msgid "jsonfasprovider, ssl, returned errors from send_request: %s"
msgstr ""
"jsonfasprovider, ssl, a renvoyé une erreur à partir de send_request : %s"

#: fedora/tg/identity/jsonfasprovider2.py:181
#, python-format
msgid "jsonfasprovider returned errors from send_request: %s"
msgstr "jsonfasprovider a retourné une erreur à partir de send_request : %s"

#: fedora/tg/templates/genshi/login.html:10
msgid "Log In"
msgstr "Connexion"

#: fedora/tg/templates/genshi/login.html:14
msgid "User Name:"
msgstr "Nom d'utilisateur :"

#: fedora/tg/templates/genshi/login.html:17
msgid "Password:"
msgstr "Mot de passe :"

#: fedora/tg/templates/genshi/login.html:21
#: fedora/tg/templates/genshi/login.html:84
#: fedora/tg2/templates/mako/login.mak:56
msgid "Login"
msgstr "Se connecter"

#: fedora/tg/templates/genshi/login.html:28
msgid "CSRF attacks"
msgstr "Attaques CSRF"

#: fedora/tg/templates/genshi/login.html:29
msgid ""
" are a means for a malicious website to make a request of another\n"
"        web server as the user who contacted the malicious web site.  The\n"
"        purpose of this page is to help protect your account and this server\n"
"        from attacks from such malicious web sites.  By clicking below, you are\n"
"        proving that you are a person rather than just the web browser\n"
"        forwarding your authentication cookies on behalf of a malicious\n"
"        website."
msgstr ""
" moyen pour un site malicieux de faire une requête vers un autre serveur\n"
"        en se faisant passer pour l'utilisateur qui a joint le site. Le but de\n"
"        cette page est de vous aider à protéger votre compte et le serveur\n"
"        des attaques de ces sites malicieux. En cliquant ci-dessous, vous\n"
"        prouvez que vous êtes une personne et non pas un simple navigateur\n"
"        qui relaie le cookie d'authentification pour le compte d'un site malicieux."

#: fedora/tg/templates/genshi/login.html:36
msgid "I am a human"
msgstr "Je suis un humain"

#: fedora/tg/templates/genshi/login.html:41
msgid "Forgot Password?"
msgstr "Mot de passe oublié ?"

#: fedora/tg/templates/genshi/login.html:42
msgid "Sign Up"
msgstr "S'enregistrer"

#: fedora/tg/templates/genshi/login.html:60
msgid "Welcome"
msgstr "Bienvenue"

#: fedora/tg/templates/genshi/login.html:82
#: fedora/tg2/templates/mako/login.mak:54
msgid "You are not logged in"
msgstr "Vous n'êtes pas connecté"

#: fedora/tg/templates/genshi/login.html:89
#: fedora/tg2/templates/mako/login.mak:62
msgid "CSRF protected"
msgstr "Protégé contre les CSRF"

#: fedora/tg/templates/genshi/login.html:91
#: fedora/tg2/templates/mako/login.mak:65
msgid "Verify Login"
msgstr "Vérifier l'identifiant"

#: fedora/tg/templates/genshi/login.html:97
#: fedora/tg2/templates/mako/login.mak:72
msgid "Logout"
msgstr "Se déconnecter"

#: fedora/tg/visit/jsonfasvisit1.py:107 fedora/tg/visit/jsonfasvisit2.py:136
#, python-format
msgid "updating visit (%s)"
msgstr "Mise à jour de la visite (%s)"

#: fedora/tg2/templates/mako/login.mak:33
msgid "Logged in:"
msgstr "Connecté : "

#: fedora/wsgi/csrf.py:113
msgid "Creating CSRFProtectionMiddleware"
msgstr "Création de CSRFProtectionMiddleware"

#: fedora/wsgi/csrf.py:122
#, python-format
msgid "clean_environ(%s)"
msgstr "clean_environ(%s)"

#: fedora/wsgi/csrf.py:125
#, python-format
msgid "Deleting %(key)s from environ"
msgstr "Suppression de %(key)s de l'environnement"

#: fedora/wsgi/csrf.py:138
#, python-format
msgid "CSRFProtectionMiddleware(%(r_path)s)"
msgstr "CSRFProtectionMiddleware(%(r_path)s)"

#: fedora/wsgi/csrf.py:145
msgid "User supplied CSRF token matches environ!"
msgstr "Le jeton CSRF fourni par l'utilisateur correspond !"

#: fedora/wsgi/csrf.py:148
msgid "Clearing identity"
msgstr "Retour - Identité"

#: fedora/wsgi/csrf.py:156
#, python-format
msgid ""
"Invalid CSRF token.  User supplied (%(u_token)s) does not match what's in "
"our environ (%(e_token)s)"
msgstr ""
"Jeton CSRF invalide. L'utilisateur a fourni (%(u_token)s) qui ne correspond "
"pas à l'environnement (%(e_token)s)"

#: fedora/wsgi/csrf.py:165
msgid "CSRF_AUTH_STATE; rewriting headers"
msgstr "CSRF_AUTH_STATE; réécriture des entêtes"

#: fedora/wsgi/csrf.py:171
#, python-format
msgid "response.location = %(r_loc)s"
msgstr "response.location = %(r_loc)s"

#: fedora/wsgi/csrf.py:246
#, python-format
msgid "CSRFMetadataProvider.add_metadata(%(r_path)s)"
msgstr "CSRFMetadataProvider.add_metadata(%(r_path)s)"

#: fedora/wsgi/csrf.py:252
#, python-format
msgid "session_id = %(s_id)r"
msgstr "session_id = %(s_id)r"

#: fedora/wsgi/csrf.py:259
msgid "Identity updated with CSRF token"
msgstr "Identité mise à jour à partir du jeton CSRF"

#: fedora/wsgi/csrf.py:262
msgid "Setting CSRF_AUTH_STATE"
msgstr "Configuration CSRF_AUTH_STATE"

#: fedora/wsgi/csrf.py:272
msgid "Got HTTPFound(302) from repoze.who.application"
msgstr "Reçu HTTPFound(302) de repoze.who.application"

#: fedora/wsgi/csrf.py:285
#, python-format
msgid "Altered headers: %(headers)s"
msgstr "Entêtes modifiés : %(headers)s"

#: fedora/wsgi/csrf.py:288
#, python-format
msgid "Invalid session cookie %(s_id)r, not setting CSRF token!"
msgstr "Cookies %(s_id)r de session invalides. Jeton CSRF non défini !"

#: fedora/wsgi/csrf.py:298
#, python-format
msgid "%(token)s in GET"
msgstr "%(token)s en GET"

#: fedora/wsgi/csrf.py:306
#, python-format
msgid "%(token)s in POST"
msgstr "%(token)s en POST"

#: fedora/wsgi/faswho/faswhoplugin.py:144
msgid "Keep alive cache miss"
msgstr "Envoi d'une commande anti-déconnexion"

#: fedora/wsgi/faswho/faswhoplugin.py:158
msgid "in identify()"
msgstr "dans identify()"

#: fedora/wsgi/faswho/faswhoplugin.py:165
#, python-format
msgid "Request identify for cookie %(cookie)s"
msgstr "Requête d'identification pour le cookie %(cookie)s"

#: fedora/wsgi/faswho/faswhoplugin.py:189
msgid "In remember()"
msgstr "Dans remember()"

#: fedora/wsgi/faswho/faswhoplugin.py:202
msgid "In forget()"
msgstr "Dans forget()"

#: fedora/wsgi/faswho/faswhoplugin.py:209
#, python-format
msgid "Forgetting login data for cookie %(s_id)s"
msgstr "Oubli des données de connexion pour le cookie %(s_id)s"

#: fedora/wsgi/faswho/faswhoplugin.py:226
msgid "In authenticate()"
msgstr "Dans authenticate()"

#: fedora/wsgi/faswho/faswhoplugin.py:250
msgid "Authentication failed, setting error"
msgstr "Authentification échouée, erreur de configuration"

#: fedora/wsgi/faswho/faswhoplugin.py:271
msgid "An unknown error happened when trying to log you in. Please try again."
msgstr ""
"Une erreur non identifiée est intervenue en tentant de vous connecter. Merci"
" de réessayer."

#: fedora/wsgi/faswho/faswhoplugin.py:282
msgid "Metadata cache miss - refreshing metadata"
msgstr "Nettoyage des méta données expirées dans le cache"

#: fedora/wsgi/faswho/faswhoplugin.py:313
msgid "In add_metadata"
msgstr "Dans add_metadata"

#: fedora/wsgi/faswho/faswhoplugin.py:317
msgid "Error exists in session, no need to set metadata"
msgstr ""
"Une erreur existe dans la session. Pas besoin de définir les métadonnées"

#: fedora/wsgi/faswho/faswhoplugin.py:327
#, python-format
msgid "Request metadata for cookie %(cookie)s"
msgstr "Requête de métadonnées pour le cookie %(cookie)s"

#: fedora/wsgi/test/controllers/root.py:31
msgid "Wrong credentials"
msgstr "Mauvaises accréditations"

#: fedora/wsgi/test/controllers/root.py:46
#, python-format
msgid "Welcome back, %s!"
msgstr "Bon retour %s !"

#: fedora/wsgi/test/controllers/root.py:56
msgid "We hope to see you soon!"
msgstr "Nous espérons vous revoir bientôt !"


